code/game/machinery/computer/abnormality_work.dm

```diff
@@ -110,8 +110,9 @@
 	update_icon()
 	working = TRUE
 	var/work_chance = datum_reference.get_work_chance(work_type, user)
+	work_chance *= user.physiology.work_success_mod // Applies Pre-Temperence now.
 	work_chance += get_attribute_level(user, TEMPERANCE_ATTRIBUTE) / 5 // For a maximum of 26 at 130 temperance
-	work_chance = clamp(work_chance * user.physiology.work_success_mod, 0, 100)
+	work_chance = clamp(work_chance, 0, 100)
```

Comment:
This is not needed in general, since prob does this automatically, as far as I am aware.